test/functional/terminal/tui_spec.lua

```diff
@@ -831,6 +831,22 @@ describe('TUI', function()
       {3:-- TERMINAL --}                                    |
     ]=])
   end)
+
+  it('forwards unknown DCS/OSC sequences to outer terminal', function()
+    local received
+
+    screen._handle_term_unhandled = function(command)
+      received = command
+    end
+
+    local unknown_dcs_seq = '\033Pblah\033\\'
+    feed_data(unknown_dcs_seq)
+    eq(unknown_dcs_seq, received)
```

Comment:
You must run the event loop in order for handlers to be called. So this needs to be 
```
screen:expect{condition=function()
  eq(unknown_dcs_seq, received)
end}
```